tests/test_cos_integration.py

```diff
@@ -38,86 +47,57 @@ async def test_build_deploy_knative_charms(ops_test: OpsTest):
     )
 
     await ops_test.model.deploy(
-        knative_charms["knative-serving"],
-        application_name="knative-serving",
-        config={"namespace": "knative-serving", "istio.gateway.namespace": ops_test.model_name},
+        # knative_charms["knative-serving"],
+        # application_name="knative-serving",
+        "knative-serving",
+        channel="latest/edge",
+        config={
+            "namespace": f"{ops_test.model_name}-serving",
+            "istio.gateway.namespace": ops_test.model_name,
+        },
         trust=True,
     )
 
     await ops_test.model.deploy(
-        knative_charms["knative-eventing"],
-        application_name="knative-eventing",
-        config={"namespace": "knative-eventing"},
+        # knative_charms["knative-eventing"],
+        # application_name="knative-eventing",
+        "knative-eventing",
+        channel="latest/edge",
+        config={"namespace": f"{ops_test.model_name}-eventing"},
```

Comment:
You are right, I was testing it locally without building them and forgot to remove it.
BTW, what do you think about using `f"{ops_test.model_name}-eventing"` instead of `"knative-eventing"` namespace? I did this to not break my current CKF deployment.

Reply:
I 'm not sure because we apply those:
* knative-eventing [namespace](https://github.com/canonical/knative-operators/blob/main/charms/knative-eventing/src/manifests/Namespace.yaml.j2#L4) from config https://github.com/canonical/knative-operators/blob/main/charms/knative-eventing/src/charm.py#L132
* knative-serving [namespace](https://github.com/canonical/knative-operators/blob/main/charms/knative-serving/src/manifests/Namespace.yaml.j2#L4) also from config https://github.com/canonical/knative-operators/blob/main/charms/knative-serving/src/charm.py#L169

Reply:
I saw that, so manifests are rendered with any namespace. What do you say to keep it here, since tests are working fine?

Reply:
ok let's keep it